api/controllers/requestHandlers/handlePathwayAnalysis.js

```diff
@@ -3,7 +3,12 @@ var dbutil = require('../utils/dbutil')
 var genedesc = require('../utils/genedesc')
 var quicksortobj = require('../utils/quicksortobj')
 
-var Queue = kue.createQueue() // this does not create a new queue, it's just redis access
```

Comment:
need to test if this would break anything

Reply:
Should I change it back? Without this change docker won't work

Reply:
I will test this locally and get back to you